src/index.html

```diff
@@ -9,6 +9,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <!--Logo-->
+      <a href="#home">
+        <img src="images/logo.png" alt="Logo">
+      </a>
+      <!--Main Nav-->
+      <nav class="navigation">
+        <ul class="main-nav">
+          <li>
+            <a class="nav-link is-active" 
+              href="#apple">APPLE</a>
+          </li>
+          <li>
+            <a class="nav-link spaceBeetween" 
+              href="#samsung">SAMSUNG</a>
+          </li>
+          <li>
+            <a class="nav-link spaceBeetween" 
+              href="#smartphone">SMARTPHONE</a>
+          </li>
+          <li>
+            <a class="nav-link spaceBeetween" 
+              data-qa="hover" 
+              href="#laptop">LAPTOPS & COMPUTERS</a>
```

Comment:
Need to fix it everywhere
```suggestion
            <a
              class="nav__link" 
              data-qa="hover" 
              href="#laptop"
            >
              LAPTOPS & COMPUTERS
            </a>
```